# VelSensorSetup block
# Block_Number 4244

Header      header

# SBF block header including time header
BlockHeader block_header

uint8   port
float32 lever_arm_X
float32 lever_arm_Y
float32 lever_arm_Z